src/report.rs

```diff
@@ -268,26 +291,45 @@ mod protobuf {
                         locs.push(function_id);
                     }
                 }
+                let thread_name = protos::Label {
+                    key: *strings.get(THREAD).unwrap() as i64,
+                    str: *strings.get(&key.thread_name_or_id().as_str()).unwrap() as i64,
+                    ..protos::Label::default()
+                };
                 let sample = protos::Sample {
                     location_id: locs,
-                    value: vec![*count as i64],
+                    value: vec![
+                        *count as i64,
+                        *count as i64 * 1_000_000_000 / self.timing.frequency as i64,
+                    ],
+                    label: vec![thread_name],
                     ..protos::Sample::default()
```

Comment:
All values have been specified. Please remove this line.

Reply:
Done. Thank you.